runelite-client/src/main/java/net/runelite/client/plugins/microbot/util/bank/Rs2Bank.java

```diff
@@ -1090,31 +1090,64 @@ public static BankLocation getNearestBank() {
      * @return true if player location is less than 4 tiles away from the bank location
      */
     public static boolean walkToBank() {
+        return walkToBank(getNearestBank());
+    }
+
+    /**
+     * Walk to bank location
+     * 
+     * @param bankLocation 
+     * @return true if player location is less than 4 tiles away from the bank location
+     */
+    public static boolean walkToBank(BankLocation bankLocation) {
```

Comment:
Why do we have to pass a banklocation? shouldn't we by default take the closest available bank?

Reply:
See the above comment...